.vscode/launch.json

```diff
@@ -4,6 +4,16 @@
     // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
     "version": "0.2.0",
     "configurations": [
+        {
+            "type": "node",
+            "request": "launch",
+            "name": "Launch Program",
+            "program": "${workspaceFolder}/src/main.js",
+            "args": [
+                "--dev",
+                "--port=8040"
+            ],
+        },
         {
```

Comment:
This starts the server under the debugger on the same port that webapp expects to find it. Useful when wanting to debug webapp requests.